### ISSUE TITLE
issue with submod duplication

### ISSUE THREAD

**Issue by chaeyoungshiie** [Monika-After-Story/MonikaModDev#9342]
this afternoon while adding new submods i believe i accidentally duplicated them while moving the files around...
just for context: yes, i have checked if i have multiple game folders and i do not, i cannot find duplicates of any of my submods anywhere, and yes it happens with all of my submods by the look of it except dialogue ones. even after removing the current submod that shows in the traceback text below, the same thing happens with another submod instead. cause like i explained, it happens with all of my submods except dialogue ones.
this is the error message i get:

I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/Submods/Kventis_RPC/kventis_rpc.54bf375dcd945403.rpy", line 14, in <module>
    version_updates={}
  File "game/zz_submods.rpy", line 118, in __init__
SubmodError: A submod with name 'Discord RPC' has been installed twice. Please, uninstall the duplicate.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "lib/windows-i686/Submods/Kventis_RPC/kventis_rpc.54bf375dcd945403.rpyc", line 6, in script
  File "C:\Users\(user)\Desktop\Just Monika\renpy\ast.py", line 814, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "C:\Users\(user)\Desktop\Just Monika\renpy\python.py", line 1719, in py_exec_bytecode
    exec bytecode in globals, locals
  File "game/Submods/Kventis_RPC/kventis_rpc.54bf375dcd945403.rpy", line 14, in <module>
    version_updates={}
  File "game/zz_submods.rpy", line 118, in __init__
SubmodError: A submod with name 'Discord RPC' has been installed twice. Please, uninstall the duplicate.

Windows-8-6.2.9200
Ren'Py 6.99.12.4.2187
Monika After Story 0.12.9

someone please help me fix it! ive been trying for the past hours and due to my anxiety and slight ocd i cant help but only have panic attacks over this submod issue...
(yes i am willing to show sc of my folders if needed!)

**Comment by Booplicate** [comment]
Generally submod issues should be reported back to the submod creators.

Show your base directory, `game/` folder, and `Submods/` folder.

**Comment by chaeyoungshiie** [comment]
hello! yes i thought of that but sadly since it happens to all my submods i dont really know who id have to report it to.
here are the screenshots, i will send more if needed. i sent them in order of the folders you asked
![sc1](https://user-images.githubusercontent.com/109702276/180097981-0074db9e-b73b-4ef4-a71f-523d40683353.png)
![sc2](https://user-images.githubusercontent.com/109702276/180097994-16f66a29-2c46-4670-a56c-7f3503bd7803.png)
![sc3](https://user-images.githubusercontent.com/109702276/180098002-8b1adb62-3f69-47c4-88ee-c2eb59196fb5.png)

**Comment by Booplicate** [comment]
That's not everything. Also what's that `rpc` folder in the base directory?

**Comment by chaeyoungshiie** [comment]
hi! sorry for late reply since i was not awake. yes i did actually, ive tried removing the sub mod and whenever i do the exact same error message pops up but with the Extra Everything submod instead, and if i remove the Extra Everything submod the same thing happens, so basically it’s really on all of my submods.

**Comment by chaeyoungshiie** [comment]
> That's not everything. Also what's that `rpc` folder in the base directory?

oh sorry. which folder did i not completely show?
